inc/Addon/ServiceProvider.php

```diff
@@ -75,17 +70,35 @@ protected function addon_cloudflare( Options_Data $options ) {
 			return;
 		}
 
+		$cf_api_key = defined( 'WP_ROCKET_CF_API_KEY' ) ? rocket_get_constant( 'WP_ROCKET_CF_API_KEY' ) : $options->get( 'cloudflare_api_key', '' );
+
+		$this->provides[] = 'cloudflare_auth';
+		$this->provides[] = 'cloudflare_client';
+		$this->provides[] = 'cloudflare_endpoints';
+		$this->provides[] = 'cloudflare';
 		$this->provides[] = 'cloudflare_subscriber';
+		$this->provides[] = 'cloudflare_admin_subscriber';
```

Comment:
@Tabrisrp we could extract cloudflare into its own serviceprovider as here we rely on the sucuri subscriber to execute that logic. It works but it is not that clean either.